Content.Server/Disease/Effects/DiseaseProgression.cs

```diff
@@ -0,0 +1,27 @@
+using Content.Server.Disease.Components;
+using JetBrains.Annotations;
+using Content.Shared.Disease;
+
+namespace Content.Server.Disease.Effects
+{
+    /// <summary>
+    /// Handles any disease with stages
```

Comment:
I am also going to say that needing this effect to be attached to the disease in order for it to work is not intuitive and it could literally just be integrated with the normal disease system.

Reply:
The reason why I choice to use effect is leaving a possiblity that under a condition ( say under 200K ) you can alter the progression of the disease. Or that the disease progress slowly at the start , then ramp up progrssion slowly